app/src/main/java/fr/free/nrw/commons/utils/ImageUtils.java

```diff
@@ -83,33 +84,26 @@
     }
 
     /**
-     * @param bitmapRegionDecoder BitmapRegionDecoder for the image we wish to process
-     * @return IMAGE_OK if image is neither dark nor blurry or if the input bitmapRegionDecoder provided is null
+     * @return IMAGE_OK if image is not too dark
      * IMAGE_DARK if image is too dark
      */
-    static @Result
-    int checkIfImageIsTooDark(BitmapRegionDecoder bitmapRegionDecoder) {
-        if (bitmapRegionDecoder == null) {
-            Timber.e("Expected bitmapRegionDecoder was null");
-            return IMAGE_OK;
-        }
-
-        int loadImageHeight = bitmapRegionDecoder.getHeight();
-        int loadImageWidth = bitmapRegionDecoder.getWidth();
-
-        int checkImageTopPosition = 0;
-        int checkImageLeftPosition = 0;
-
-        Timber.v("left: " + checkImageLeftPosition + " right: " + loadImageWidth + " top: " + checkImageTopPosition + " bottom: " + loadImageHeight);
-
-        Rect rect = new Rect(checkImageLeftPosition,checkImageTopPosition, loadImageWidth, loadImageHeight);
+    static @Result int checkIfImageIsTooDark(String imagePath) {
+        long millis = System.currentTimeMillis();
+        try {
+            Bitmap bmp = new ExifInterface(imagePath).getThumbnailBitmap();
```

Comment:
While testing the PR, lets check if the thumbnail information is present in EXIF for images picked from Google photos.